isis/src/qisis/apps/ipce/IpceMainWindow.h

```diff
@@ -199,8 +200,8 @@ namespace Isis {
       static const int m_maxRecentProjects = 5;
 
       QToolBar *m_permToolBar; //!< The toolbar for actions that rarely need to be changed.
-      QToolBar *m_activeToolBar; //<! The toolbar for the actions of the current tool.
-      QToolBar *m_toolPad; //<! The toolbar for the actions that activate tools.
+      //QToolBar *m_activeToolBar; //<! The toolbar for the actions of the current tool.
```

Comment:
If these are commented out can they be removed?

Reply:
These are removed in another PR, since I did not want to do too much on this branch.

Reply:
Might as well remove them here, just to make sure that they are deleted. We don't want any more commented out code if we can avoid it.